analytics_data_api/constants/learner.py

```diff
@@ -0,0 +1 @@
+LEARNER_API_ROSTER_PAGE_SIZE = 25
```

Comment:
How about `LEARNER_API_DEFAULT_LIST_PAGE_SIZE`? I'm trying to consistently use the term 'list' in API lingo, and I like the idea of learner constants beginning with `LEARNER`.